shotover/src/runner.rs

```diff
@@ -171,6 +186,44 @@ impl Shotover {
     pub fn run_block(self) -> ! {
         let (trigger_shutdown_tx, trigger_shutdown_rx) = tokio::sync::watch::channel(false);
 
+        let _received_file_descriptors = if let Some(socket_path) = &self.hotreload_from_socket {
+            info!("Hot reload CLIENT mode - requesting socket handoff from existing shotover");
+
+            let client = crate::hot_reload_client::UnixSocketClient::new(socket_path.clone());
+            match self.runtime.block_on(async {
+                client
+                    .send_request(crate::hot_reload::Request::SendListeningSockets)
+                    .await
+            }) {
+                Ok(crate::hot_reload::Response::SendListeningSockets { port_to_fd }) => {
+                    info!(
+                        "Successfully received {} file descriptors for hot reload",
+                        port_to_fd.len()
+                    );
+                    for (port, fd) in &port_to_fd {
+                        info!("Received file descriptor {} for port {}", fd.0, port);
+                    }
+                    Some(port_to_fd)
+                }
+                Ok(crate::hot_reload::Response::Error(msg)) => {
+                    error!("Hot reload request failed: {}", msg);
+                    std::process::exit(1);
+                }
+                Err(e) => {
+                    error!(
```

Comment:
error logs are a bit different to println, this code would result in 3 distinct logs about 1 specific error, which is confusing.

Please make these 3 calls into a single `error!()` call, you can use `\n` or multiline strings if needed.
e.g.
```rust
error!(r#"line1
line2
line3"#);
```